shared/common-adapters/avatar.desktop.js

```diff
@@ -19,9 +19,20 @@ export default class Avatar extends Component {
     this.state = {avatarLoaded: false}
   }
 
+  _createUrl (): ?string {
+    if (this.props.url) {
+      return this.props.url
+    } else if (this.props.username) {
```

Comment:
Also usercard does `keybase.io/<username>` and that apparently works? https://github.com/keybase/client/blob/master/shared/common-adapters/user-card.desktop.js#L12